reflex/components/graphing/plotly.py

```diff
@@ -36,10 +36,10 @@ class Plotly(PlotlyLib):
     use_resize_handler: Var[bool]
 
     def _get_imports(self):
-        return {}
+        return {"next/dynamic": {ImportVar(tag="dynamic", is_default=True)}}
```

Comment:
seems like this could be exposed as a reusable name

```python
DYNAMIC_IMPORT = {"next/dynamic": {ImportVar(tag="dynamic", is_default=True)}}
```

if the "no ssr" import functionality could be baked into the existing `ImportVar` compiler mechanism, then that might be even better... but a quick look shows that is not trivial as it's currently written.

So maybe an implementation like

```python
class NoSSRComponent(Component):
    def _get_imports(self):
        return {"next/dynamic": {ImportVar(tag="dynamic", is_default=True)}}

    def _get_custom_code(self) -> str:
        return f"const {self.tag} = dynamic(() => import('{self.library}'), { ssr: false });"
```

Reply:
Good point thanks